tests/ut_s9srpcclient/ut_s9srpcclient.cpp

```diff
@@ -2175,4 +2177,22 @@ UtS9sRpcClient::testDeleteUserPreferences()
     return true;
 }
 
+bool
+UtS9sRpcClient::testDatetimeRegex() {
+    S9S_COMPARE(isValidDateTimeFormat("randomStr"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-31T14:27:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-13-31 14:27:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-32 14:27:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-1 14:27:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-10 24:27:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-10 21:60:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-10 21:06:60"), false);
+    S9S_COMPARE(isValidDateTimeFormat("aaa2020-12-31 14:27:40"), false);
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-31 14:27:40ccc"), false);
+
+    S9S_COMPARE(isValidDateTimeFormat("2020-12-31 14:27:40"), true);
+
```

Comment:
maybe to add also:
`S9S_COMPARE(isValidDateTimeFormat("2023-12-31 23:59:59"), true);`
`S9S_COMPARE(isValidDateTimeFormat("2024-01-01 00:00:00"), true);`
and, maybe, some other dates around some date/time limits/bounds.